docs/product/accounts/quotas/manage-attachments-quota.mdx

```diff
@@ -35,7 +35,7 @@ Quotas can only be updated by a Billing or Owner member of your Sentry organizat
 
 Once your attachments volume is approaching or has exceeded the quota, teammates with the "Owner" organization permission level will receive [notification](/product/alerts/notifications/#quota-notifications) emails. You can then choose to [increase](#increasing-quotas) or [decrease](#decreasing-quotas) your quota.
 
-If this is your first time exceeding quota and you're on a paid plan, however, you'll be entered into a one-time grace period. Learn more about the grace period in this [Help article](https://help.sentry.io/account/billing/what-happens-when-i-run-out-of-event-capacity-and-a-grace-period-is-triggered/).
+Unlike exceeding your error event quota for your first time exceeding quota while on a paid plan, you will not be entered into a one-time grace period the first time you deplete your attachments quota. Learn more about the grace period in this [Help article](https://sentry.zendesk.com/hc/en-us/articles/23983502355099-What-happens-when-I-run-out-of-event-capacity-and-a-grace-period-is-triggered).
```

Comment:
```suggestion

```
I think if a grace period doesn't apply to attachments quota, we should just delete this section. I think it's confusing to tell users about something that won't apply to them.